docs/network/smart-rollup-nodes.md

```diff
@@ -189,3 +189,59 @@ For example, this query gets the health of the node:
    ```
 
 Now that you have a Smart Rollup node configured for Etherlink, you can run an Etherlink EVM node, as described in [Running an Etherlink EVM node](/network/evm-nodes).
+
+## Stopping the Smart Rollup node
+
+When you want to stop running the Smart Rollup node, you must wait for the last commitment that your node made to be cemented.
+Then you can recover the node's bonded tez and stop the node.
+
+:::warning
+
+If you stop the node before waiting for its last commitment to be cemented, the bonded tez is at risk if other Smart Rollup nodes challenge that commitment and your node is not online to defend it in the Smart Rollup refutation game.
+For more information, see [Smart Rollups](https://docs.tezos.com/architecture/smart-rollups) on docs.tezos.com.
+
+:::
+
+Follow these steps to stop an Etherlink Smart Rollup node:
+
+1. Switch the node to `bailout` mode.
+A node running in `bailout` mode defends its existing commitments but does not make new commitments.
+
+   1. Stop the node.
+
+   1. Re-initialize the node for `bailout` mode by running this command:
+
+      ```bash
+      octez-smart-rollup-node init bailout config for sr1Ghq66tYK9y3r8CC1Tf8i8m5nxh8nTvZEf \
+        with operators \
+        operating:$OPERATOR_ACCOUNT \
+        cementing:$SECONDARY_ACCOUNT \
+        executing_outbox:$SECONDARY_ACCOUNT \
+        --rpc-addr 0.0.0.0 \
+        --data-dir $SR_DATA_DIR \
+        --pre-images-endpoint https://snapshots.eu.tzinit.org/etherlink-mainnet/wasm_2_0_0 \
+        --force
+      ```
+
+   1. Restart the node:
+
+      ```bash
+      octez-smart-rollup-node --endpoint $MY_LAYER_1_NODE run \
+        --data-dir $SR_DATA_DIR
+      ```
+
```

Comment:
I'm not 100% sur, but I think it would be possible to not modify the config and simply run with ``run --mode bailout` no ?

Reply:
I see, I don't need to modify the config to run in a new mode. I made updates. How does this look? Is it OK that the config will continue to show observer mode?